docs/_documentations/remote-removing.md

```diff
@@ -10,7 +10,29 @@ parent: installoncloud
 order: 1
 ---
 
-## Removing Codewind remote
+# Removing your remote deployment of Codewind
 
-To remove Codewind remote, enter the following `cwctl` command: `cwctl remove remote --namespace {mynamespace} --workspace {workspaceID}`
+The Codewind CLI `cwctl` is designed to do almost all the steps required to remove a remote deployment of Codewind by chaining together a number of Kubernetes operations.
 
+There are two modes of use. You can:
+1. Remove Codewind.
+2. Remove Keycloak. A Keycloak service can provide authentication services to many Codewind deployments in different namespaces, so only remove it if you are sure it is no longer used.
```

Comment:
remove it ~~if~~ when you are sure Keycloak is no longer being used

Reply:
Corrected